src/app/give-help/accommodation-form/accommodation-form.component.ts

```diff
@@ -22,13 +22,18 @@ export class AccommodationFormComponent {
     hostLanguage: [],
   });
   loading: boolean = false;
+  @ViewChild('phoneInput')
+  phoneInput!: { nativeElement: { value: any } };
   constructor(
     private accommodationsResourceService: AccommodationsResourceService,
     private router: Router,
     private snackbarService: SnackbarService
   ) {}
 
-  onPhoneNumberChange(): void {
+  onPhoneNumberChange($event: any): void {
+    let val = $event.target.value;
+    val = val.replace(/[^0-9 ]+/g, '');
```

Comment:
Możesz wytłumaczyć logikę tutaj?

Reply:
Blokowanie wprowadzania czegokolwiek poza cyframi,  dajmy na to jeśli próbujesz wprowadzić literę jest replace na '', blokada z perspektywy UI.

Reply:
Widzę, że spacje też są dozwolone. To celowe?

BTW, jeśli chcemy pozwalać na same cyfry, to czy input numeryczny by nie zadziałał? (tylko bez strzałek góra-dół, bo one nie mają sensu przy nrach tel.).

Reply:
słuszna uwaga, dzięki zapomniałem

Reply:
Zadziałał by, nawet go spiąłem na początku ale mam tak ze nadpisywanie defaultowych styli przeglądarki jak:
`/* Chrome, Safari, Edge, Opera */
input::-webkit-outer-spin-button,
input::-webkit-inner-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

/* Firefox */
input[type=number] {
  -moz-appearance: textfield;
}`
wydaje mis ie zawsze bardziej takim hackiem niz podejsciem, robie tam gdzie nie mozna inaczej, ale moze ma to sens tutaj jeszcze sie pochyle nad tym, dzieki.

Reply:
Poprawione